docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,307 @@
+**`ClientRootCAs`**: (Mutual TLS only)  Can be left blank if mutual TLS is disabled. If mutual TLS is enabled, this is the path to the root certificate of the TLS CA.
+
+## General.KeepAlive
+
+The `KeepAlive` values might need to be tuned for compatibility with any networking devices or software (like firewalls or proxies) in between components of your network. Ideally, these settings would be manipulated if needed in a test or pre-prod environment and then set accordingly for your production environment.
+
+```
+# ServerMinInterval is the minimum permitted time between client pings.
+# If clients send pings more frequently, the server will
+# disconnect them.
+ServerMinInterval: 60s
+# ServerInterval is the time between pings to clients.
+ServerInterval: 7200s
+# ServerTimeout is the duration the server waits for a response from
+# a client before closing the connection.
+ServerTimeout: 20s
+```
+
+**`ServerMinInterval`**: (default value should not be overridden, unless determined necessary through testing)
+**`ServerInterval`**: (default value should not be overridden, unless determined necessary through testing)
+**`ServerTimeout`**: (default value should not be overridden, unless determined necessary through testing)
+
+## General.Cluster
+
+```
+# SendBufferSize is the maximum number of messages in the egress buffer.
+# Consensus messages are dropped if the buffer is full, and transaction
+# messages are waiting for space to be freed.
+SendBufferSize: 10
+# ClientCertificate governs the file location of the client TLS certificate
+# If not set, the server General.TLS.Certificate is re-used.
+ClientCertificate:
+# If not set, the server General.TLS.PrivateKey is re-used.
+ClientPrivateKey:
+# The below 4 properties should be either set together, or be unset together.
+# If they are set, then the orderer node uses a separate listener for intra-cluster
+# communication. If they are unset, then the general orderer listener is used.
+# This is useful if you want to use a different TLS server certificates on the
+# client-facing and the intra-cluster listeners.
+
+# ListenPort defines the port on which the cluster listens to connections.
+ListenPort:
+# ListenAddress defines the IP on which to listen to intra-cluster communication.
+ListenAddress:
+# ServerCertificate defines the file location of the server TLS certificate used for intra-cluster
+# communication.
+ServerCertificate:
+# ServerPrivateKey defines the file location of the private key of the TLS certificate.
+ServerPrivateKey:
+```
+
+As of Fabric v2.3.0, if unset, the `ClientCertificate` and `ClientPrivateKey` default to the server `General.TLS.Certificate` and `General.TLS.PrivateKey`.
+
+**`ClientCertificate`**:  Provide the path to and filename of the public certificate (also known as a signed certificate) generated by your TLS CA for this node.
+**`ClientPrivateKey`**: Provide the path to and filename of the private key generated by your TLS CA for this node.
```

Comment:
This entry and others in this file are not on separate lines in the generated doc.